tools/c7n_azure/c7n_azure/constants.py

```diff
@@ -21,8 +21,6 @@
 FUNCTION_EVENT_TRIGGER_MODE = 'azure-event-grid'
 FUNCTION_TIME_TRIGGER_MODE = 'azure-periodic'
 FUNCTION_KEY_URL = 'hostruntime/admin/host/systemkeys/_master?api-version=2018-02-01'
-FUNCTION_CONSUMPTION_BLOB_CONTAINER = 'cloud-custodian-packages'
-FUNCTION_PACKAGE_SAS_EXPIRY_DAYS = 365 * 10  # 10 years
 FUNCTION_AUTOSCALE_NAME = 'cloud_custodian_default'
```

Comment:
one other I noticed in this file, its hardcoded to py 3.6, 3.7 is also available ie
'3.7': 'mcr.microsoft.com/azure-functions/python:2.0-python3.7-appservice'  but could be a separate pr

Reply:
oh, good catch. I'd do it in a separate PR when this is tested & merged